src/main/webapp/app/overview/course-statistics/course-statistics.component.ts

```diff
@@ -560,3 +580,6 @@ export class CourseStatisticsComponent implements OnInit, OnDestroy {
         return scores.get(scoreType)!;
     }
 }
+function of(undefined: undefined): any {
+    throw new Error('Function not implemented.');
+}
```

Comment:
What is this used for?

Reply:
good catch, auto importing didn't work out so well 😅